### ISSUE TITLE
Consider adding alt text

### ISSUE THREAD

**Issue by jecode93** [AbuTalha3/Portfolio-Setup#5]
https://github.com/AbuTalha3/Portfolio-Setup/blob/4198114af732fe9b82b5e73eaf51bf100065c514/index.html#L153

Consider adding the [alt] text to specify an alternate text for an image, if the image cannot be displayed. The alt attribute provides alternative information for the picture if a user cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader).

**Comment by AbuTalha3** [comment]
done
